src/cluster_legacy.c

```diff
@@ -6521,38 +6527,81 @@ int detectAndUpdateCachedNodeHealth(void) {
 }
 
 /* Replicate migrating and importing slot states to all replicas */
-void clusterReplicateOpenSlots(void) {
-    if (!server.cluster_enabled) return;
+sds clusterEncodeOpenSlotsAuxField(int rdbflags) {
+    if (!server.cluster_enabled) return NULL;
 
-    int argc = 5;
-    robj **argv = zmalloc(sizeof(robj *) * argc);
+    /* Open slots should not be persisted (nor loaded) */
```

Comment:
```suggestion
    /* Open slots should not be persisted */
```
We do want them to be loaded right? Might just be an odd word choice.

Reply:
I meant to say "not loaded from a persisted RDB but only from full sync". Will clarify